.github/workflows/release.yml

```diff
@@ -496,6 +496,22 @@ jobs:
         cd build-x64
         ../os/macosx/notarize.sh
 
+    - name: Build zip
+      run: |
+        cd build-x64
+
+        pushd _CPack_Packages/*/Bundle/openttd-*/
+        rm -f Applications
+        rm -f ../*.dmg
```

Comment:
What does this statement do? Feels out of place, as nothing following is going to use it. Left by mistake?

Reply:
Removes the Applications symlink so it doesn't end up in the zip, and removes the .dmg so it doesn't conflict when trying to expand the folder name wildcard for the zip name. Not the nicest bit of code admittedly but it works.

Reply:
I hope the following doesn't come as surprise:

clearly this has to be added in comments to the code ;) As clearly it is non-obvious what you are doing here :D

Reply:
No, not unexpected. :) Updated now.